README.md

```diff
@@ -160,6 +162,14 @@ Results for applying pruning to reduce the parameters of the following models by
 | DenseNet121 [3] | 74.44                | 71.71                 |
 
 
+## Quantization Bag of Tricks
+
+If the quantization process degrades the model's accuarcy too much, check out the [Bag of Tricks](https://github.com/sony/model_optimization/tree/main/quantization_bag_of_tricks.md)
```

Comment:
what is "too much"? I think it needs to be rephrased to something like:
"If the quantized model (suffers from a significant accuracy degradation /  accuracy results are not sufficient)..."